pkg/monitortests/testframework/watchclusteroperators/operator.go

```diff
@@ -176,19 +176,7 @@ func startClusterOperatorMonitoring(ctx context.Context, m monitorapi.RecorderWr
 			for i := range cv.Status.Conditions {
 				s := &cv.Status.Conditions[i]
 				previous := findOperatorStatusCondition(oldCV.Status.Conditions, s.Type)
-				if previous == nil {
-					continue
-				}
-				if s.Status != previous.Status {
-					var msg string
-					switch {
-					case len(s.Reason) > 0 && len(s.Message) > 0:
-						msg = fmt.Sprintf("changed %s to %s: %s: %s", s.Type, s.Status, s.Reason, s.Message)
-					case len(s.Message) > 0:
-						msg = fmt.Sprintf("changed %s to %s: %s", s.Type, s.Status, s.Message)
-					default:
-						msg = fmt.Sprintf("changed %s to %s", s.Type, s.Status)
-					}
+				if previous == nil || s.Status != previous.Status {
```

Comment:
This is to keep consistent with https://github.com/openshift/origin/blob/67efa89c27fcd01a404e5ad3c0b6521e33a74207/pkg/monitortests/testframework/watchclusteroperators/operator.go#L44